### DISCUSSION TITLE
原项目关闭了?为啥啊

### DISCUSSION THREAD

**Discussion by Vanilla-Yukirin** [Cute-Dress/Dress#29] [畅聊]
回来看看我的照片结果发现搜不到原来的Dress项目了 大哭

**Comment by CuteDress** [comment]
~~咱也不清楚，咱只是一个备份的（~~
https://gist.github.com/kagurazakayashi/44fa4ae106fa0e00b4172de7de8e69c6

**Reply by CuteDress** [reply]
另外，刚刚顺便pick了[bdef9cf](https://github.com/kagurazakamasae/Dress/commit/bdef9cf2fe95d26ea62b87b967a3a34659c44d40)

**Comment by SnowWindSaveYou** [comment]
同好奇

**Comment by Vanilla-Yukirin** [comment]
来自`神楽坂雅詩`，2023年10月6日：
> 项目创建者出于各种原因不想维护了，大概和人越来越多越来越乱有关系，所以项目被创建者直接关掉了…… 这里是关闭前最后的备份： https://github.com/kagurazakamasae/Dress
@SnowWindSaveYou @CuteDress
